cli/serve/serve.go

```diff
@@ -244,9 +248,55 @@ func serverMain(args []string, c cli.Config) error {
 		log.Info("Now listening on ", addr)
 		return http.ListenAndServe(addr, nil)
 	}
+	if conf.MutualTLSCAFile != "" {
+		mutualTLSCAs, err := ioutil.ReadFile(conf.MutualTLSCAFile)
```

Comment:
You should use [helpers.LoadPEMCertPool](https://godoc.org/github.com/cloudflare/cfssl/helpers#LoadPEMCertPool), 

```Go
clientPool, err := helpers.LoadPEMCertPool(conf.MutualTLSCAFile)
if err != nil {
	return fmt.Errorf("failed to load mutual TLS CA file: %s", err)
}
```